content/en/agent/basic_agent_usage/aix.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Basic Agent Usage for AIX
+kind: documentation
+further_reading:
+---
+
+<div class="alert alert-info">
+The Datadog UNIX agent is being developed for specific system architectures, and is not the same as Agents 5 and 6.
+</div>
+
+This page outlines the installation and configuration of the Datadog UNIX Agent for AIX.
+
+**Note:** The Datadog UNIX Agent supports AIX 6.1, 7.1 and 7.2.
+
+## Installation
+
+Download links for the latest releases can be found on [this page][1].
+
+The installer may be executed as follows (as root):
+
+```shell
+installp -aXYgd ./datadog-unix-agent-<version>.powerpc.bff -e dd-aix-install.log datadog-unix-agent
+```
+
+This installs the agent in `/opt/datadog-agent`.
+
+Note: Agent installation logs can be found in the `dd-aix-install.log` file. To disable this logging, remove the `-e` parameter in the installation command.
+
+
+## Commands
+
+| Description                        | Command (as root)                   |
+| --------------------               | --------------------                |
+| Start Agent as a service           | `startsrc -s datadog-agent`         |
+| Stop Agent running as a service    | `stopsrc -s datadog-agent`          |
```

Comment:
I feel like we need to add a caveat note regarding stoppage of the service. 

One of the decisions we had to make was what restart policy to set on the `datadog-agent`. We decided to go for a `respawn` policy in the inittab. What this implies is that when the agent is stopped, the rudimentary init system will restart it automatically, even if it was stopped with `stopsrc` manually and intentionally.

We feel this is the ideal strategy, because if the agent were to die for any reason, it's probably best if it comes back up automatically to avoid having the customers go blind all of a sudden with no self-recovery.

If the customers want to change this behavior they can do so, but they'll have to manually edit the inittab entry for `datadog-agent`. For instance:

```
        chitab "datadog-agent:2:once:/usr/bin/startsrc -s datadog-agent"
```
By changing the "spawn" mode to `once`, if the agent is stopped, or dies, no attempts will be made to bring it back up.

The general format of inittab entries is:
```
        <service_name>:<runlevels>:<respawn|once|wait>:<command>
```

The default way we define the service behavior with is (done automatically by the install scripts):
```
        mkitab "${SERVICE_NAME}:2:respawn:/usr/bin/startsrc -s ${SERVICE_NAME}"
```

Customers can also add the service to more runlevels if they wish to do so (2 is the most common).

Let me know if you have any more questions.

Reply:
Thanks @truthbk, some followup questions:

1. Do we employ a similar "restart if stopped" strategy on our other agents?
2. Would manual modications to inittab by users persist over agent upgrades?

Reply:
Sorry for the slow response to this:
1. Yes, we automatically restart the agent in the event of a crash. The reason for this is we don't want to leave our customers blind if we have an issue on the agent side. Tools like `systemd` are a lot more elaborate and they will give up if the agent crashes incessantly, and they wouldn't restart the process if it has been shut down manually. Unfortunately AIX is a lot more rudimentary. 

2. That's a great question. Right now the installer scripts would overwrite the entry, but we could probably find a way to make it persist over agent upgrades. This could be handled by the scripts on the upgrading version, so we have time to make a decision for 1.1.0 or whatever.